micrometer-core/src/test/java/io/micrometer/core/instrument/MeterRegistryLoggingTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2024 VMware, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.core.instrument;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import com.innoq.junit.jupiter.logging.Logging;
+import com.innoq.junit.jupiter.logging.LoggingEvents;
+import io.micrometer.core.instrument.config.MeterFilter;
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+import org.assertj.core.api.InstanceOfAssertFactories;
+import org.junit.jupiter.api.Test;
+import org.slf4j.LoggerFactory;
+
+import static org.assertj.core.api.Assertions.as;
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Tests for expected logging from {@link MeterRegistry}.
+ */
+@Logging
+class MeterRegistryLoggingTest {
```

Comment:
Should we/can we add an extra test that checks the whole stacktrace (with multiple elements)?

Reply:
I updated the test to check for the 3 lines of the stack trace within the test code (what would be equivalent to user code in a real scenario). Do you think that's sufficient?

Reply:
Yes, looks great!